src/tox_poetry.py

```diff
@@ -35,10 +36,15 @@ def tox_testenv_install_deps(venv, action):
 
     project_root = venv.envconfig.config.toxinidir
 
-    cmd = [
-        venv.getcommandpath('poetry', venv=False),
-        'install',
-    ]
+    cmd = []
+    try:
+        cmd.append(venv.getcommandpath('poetry', venv=False))
+    except:
```

Comment:
Can you specify particular exception type?

Reply:
Updated to catch InvocationError given [this block of code](https://github.com/tox-dev/tox/blob/6305f1821eef8eef463ae86eb9ea06e73975dd44/src/tox/venv.py#L193).